documentation/cubic-pos-article.md

```diff
@@ -0,0 +1,156 @@
+	- Slashes are properly handled to ensure that only tokens that contributed to a validator's voting power at the time of infraction are affected
+
+**Validator Sets**
+
+Namada validators are classified into one of three sets:
+- **Consensus:** validators that are active in the consensus voting mechanism, chosen to be the `max_validators` largest validators weighted by their voting power. They are the only validators who can earn block rewards from the PoS inflation.
+- **Below capacity:** validators whose bonded stake is less than the *consensus* validators but above some small threshold `min_validator_stake`.
+- **Below threshold:** all other validators with stake below `min_validator_stake`.
+
+The *consensus* and *below capacity* validator sets are explicitly held in storage, ordered by their bonded stake, and are updated at every epoch boundary. However, no ordered set of *below threshold* validators is kept in storage. This validator set construction and the `min_validator_stake` threshold exist primarily to prevent the possibility of significant slowing of PoS system updates due to spam validator account creation. Thus, unbounded iteration when updating and validating PoS system state changes is avoided. Conversely, the *below capacity* validator set must be kept in storage, ordered by bonded stake, in order to properly process validator set changes in which validators may drop out or enter the *consensus* validator set.
+
+**Important validator set configuration values, mutable via governance (with default values):**
+- `max_validators` = 100
+- `min_validator_stake` = 1 NAM
+
+# Cubic Slashing
+
+As in many other PoS blockchains, slashing is applied as a way to punish misbehaving validators. If a validator is detected to have misbehaved then portions of the validator's bonded stake will be slashed or seized. This propagates down to the stake of delegators associated to the misbehaving validator as well. Validator infractions that are subject to slashing include:
+
+- proposing an invalid block
+- signing an invalid block
+- proposing two different blocks at the same height
+- double-signing a block
+
+Ultimately, slashed tokens are sent to a slash pool fund, from which the funds can be spent by governance to refund slashed accounts in the event of accidental misbehavior.
+
+Typically, the slash amount for an infraction is proportional to a validator's voting power, however  Namada employs a slashing scheme that has more severe punishments for correlated validator misbehavior: so-called [*cubic slashing*](https://specs.namada.net/economics/proof-of-stake/cubic-slashing.html), wherein the slashed token amount can be proportional to the cube of a validator's voting power (the slash rate is quadratic). Cubic slashing employs more severe punishments to validators who commit infractions close to each other in time, making it riskier for an entity to operate larger or multiple similarly configured validators.
```

Comment:
> making it riskier for an entity to operate larger or multiple similarly configured validators

I'd say that rather it is to have very little punishment for accidental (e.g. operational) misbehavior, but much more severe for deliberate repeated attacks

Reply:
thx, changed the sentence to:
"Cubic slashing is designed to impose more severe punishments for deliberate, repeated validator attacks but much milder punishments for accidental or non-malicious misbehaviors."